docs/installation_linux.rst

```diff
@@ -46,6 +46,12 @@ Run the command below to install the dependencies on Ubuntu:
 
    sudo apt install git gcc libgirepository1.0-dev libcairo2-dev qemu-utils libvirt-dev python3-pip python3-venv -y
 
+On Azure Linux (Mariner) 2.0 or 3.0, run the following command:
```

Comment:
This document mentions `Ubuntu` in several places, please double check every place, if it needs to be updated.